.github/workflows/reusable-build-image.yml

```diff
@@ -0,0 +1,194 @@
+name: Build and push images
+
+defaults:
+  run:
+    shell: bash
+
+on:
+  workflow_call:
+    inputs:
+      image_name:
+        description: The full base name of the image e.g. ghcr.io/XRPLF/rippled
+        type: string
+        required: true
+      publishing_repo:
+        description: The repository that is allowed to publish.
+        type: string
+        required: true
+      dockerfile:
+        description: Alternative Dockerfile to use.
+        type: string
+        default: docker/Dockerfile
+        required: false
+      latest_branch:
+        description: The branch that receives the "latest" tag.
+        type: string
+        default: master
+        required: false
+      push_images:
+        description: Whether to push images to registry.
+        type: boolean
+        default: true
+        required: false
+
+jobs:
+  build:
+    name: ${{ matrix.arch }} image build
+    if: github.repository == inputs.publishing_repo
+    strategy:
+      matrix:
+        arch: [amd64, arm64]
+    runs-on: ${{ matrix.arch == 'arm64' && 'ubuntu-24.04-arm' || 'ubuntu-24.04' }}
+    permissions:
+      packages: write
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Login to GitHub Container registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Lowercase repo for image name
+        run: |
+          GITHUB_REPO=${{ inputs.image_name }}
+          echo lowercase_repo=${GITHUB_REPO@L} >> $GITHUB_ENV
+
+      - name: Prepare image metadata
+        id: meta
+        uses: docker/metadata-action@v5
+        with:
+          images: ${{ env.lowercase_repo }}
+          labels: |
+            org.opencontainers.image.authors=For inquiries, please use https://github.com/${{ github.repository }}/issues
```

Comment:
Doesn't look like `authors`

Reply:
Copypasta from XRPLF/ci where we leaned into "freeform."
[org.opencontainers.image.authors](https://specs.opencontainers.org/image-spec/annotations/#pre-defined-annotation-keys) 
> contact details of the people or organization responsible for the image (freeform string)